gulpfile.js

```diff
@@ -608,21 +601,24 @@ function enableLocalTesting(targetFile) {
 }
 
 /**
- * Main build
+ * Performs the build steps for gulp build and gulp watch
+ * @param {boolean} watch
  * @return {!Promise}
  */
-function build() {
-  process.env.NODE_ENV = 'development';
-  printConfigHelp('gulp build');
-  return compileCss().then(() => {
+function performBuild(watch) {
+  if (!watch) {
```

Comment:
Does anyone remember why this is?

Reply:
Perhaps this too was an artifact of someone enabling it for `build()` but not `watch()`? Investigating...

Reply:
Yes, it was added in #678 to `build()`, but not to `watch()`. I think it's safe to do this in either case.

Reply:
Done.